In [1]:
count = 0
with open('../../code/2/popular-names.txt') as f:
    for line in f:
        count+= 1

print (count)

2780


Linuxコマンド
wc code/2/popular-names.txt

実行結果
2780   11120   57806 code/2/popular-names.txt